app/javascript/Pages/Home.jsx

```diff
@@ -1,30 +1,38 @@
-import React from "react";
+import React, { Component } from "react";
 import Layout from "./Layout";
+import Navbar from "./components/Navbar/Navbar";
+import Footer from "./components/Footer/Footer";
 import { Head, Link } from "@inertiajs/inertia-react";
+import { Inertia } from "@inertiajs/inertia";
 import "../stylesheets/body.css";
+import "../stylesheets/Home.css";
 
-export default function Welcome(props) {
-  const token = document.querySelector(
-    "[name=csrf-token]"
-  ).content;
-  const headers = { "X-CSRF-Token": token };
+class Homepage extends Component {
+  render() {
+    const token = document.querySelector("[name=csrf-token]").content;
+    const headers = { "X-CSRF-Token": token };
 
-  return (
-    <Layout current_user={props.current_user}>
-      <Head title="Welcome" />
-      <h1>Welcome {props.current_user.name}!</h1>
-      <p>Hello, welcome to your first Inertia app!</p>
-      <p id="p1">This is cool!</p>
-      <p className="paragraph">This is cool!</p>
+    return (
+      <main>
+        <Navbar current_user={this.props.current_user} />
 
-      <Link
-        href="/logout"
-        method="delete"
-        headers={headers}
-        as="button"
-      >
-        Logout
-      </Link>
-    </Layout>
-  );
+        <img
+          src={require("../images/banner.svg")}
+          alt="banner"
+          className="hero-img"
+        />
+        <Head title="Welcome" />
+        <p>Hello, welcome to your first Inertia app!</p>
+        <p id="p1">This is cool!</p>
```

Comment:
**issue (blocking)**: don't use ids to add styles to elements. You would need 256 classes to override a single id 😱 https://ryanseddon.com/css/extreme-specificity/
Ids should only be used as anchors since in that case we do want there to only be one instance of each, everything else should use classes

Reply:
This is actually my fault, I was only trying to test if the css was working, way back to when I was setting up Inertia

Reply:
there is no fault in a team 😛 there are only improvements